app/models/task.rb

```diff
@@ -0,0 +1,11 @@
+class Task < ApplicationRecord
+  belongs_to :project
+  validates :project, presence: true
+  validates :title, presence: true
+  validate :deadline_after_creation
+
+  private
+  def deadline_after_creation
+    errors.add(:deadline_at, "must be after creation time") unless (!created_at.nil? and created_at < deadline_at) or Time.now.to_datetime < deadline_at
```

Comment:
I have a task created 2 weeks ago that had a deadline a week ago. What happens if I want to update it?

Reply:
And use `&&` and `||` instead of `and` and `or`

Reply:
And it is really long line, maximum is 120 symbols.